test/test_ddp.py

```diff
@@ -19,10 +22,13 @@ def _ddp_correctness(rank):
           'Default device {} is not a TPU device'.format(device),
           file=sys.stderr)
       return
-    util.ddp_correctness(None)
+    util.ddp_correctness(None, use_large_net=use_large_net, debug=debug)
 
   def test_ddp_correctness(self):
-    xmp.spawn(self._ddp_correctness, args=())
+    xmp.spawn(self._ddp_correctness, args=(False, FLAGS.debug))
+
+  def test_ddp_correctness_large_net(self):
+    xmp.spawn(self._ddp_correctness, args=(True, FLAGS.debug))
```

Comment:
I'm a little concerned that `FLAGS.debug` is defined and used in `distributed_util.py`, but it's referenced here. But the command line args don't seem to get propagated to child processes with `spawn`, so you can't reference `FLAGS.debug` there. Is it possible to pass the whole `FLAGS` object to the child processes instead? That's what we do in `test_train_mnist_mp.py` for example.

Reply:
Good point. Let me fix that.

Reply:
It looks like `absl.flags` can't be pickled... Let me try using `args_parse`.